adserver/templates/adserver/reports/advertiser-keyword.html

```diff
@@ -0,0 +1,38 @@
+{% extends "adserver/reports/advertiser.html" %}
+{% load humanize %}
+{% load i18n %}
+{% load metabase %}
+
+
+{% block title %}{% trans 'Advertiser Keyword Report' %} - {{ advertiser }}{% endblock %}
+
+
+{% block heading %}
+{% blocktrans %}Advertiser Keyword Report for {{ advertiser }}{% endblocktrans %}
+{% endblock heading %}
+
+{% block breadcrumbs %}
+  {{ block.super }}
+  <li class="breadcrumb-item active">{% trans 'Keyword Report' %}</li>
+{% endblock breadcrumbs %}
+
+
+{% block explainer %}
+<section class="mb-5">
+  <h3>{% trans 'About this report' %}</h3>
+  <p>{% trans 'This report shows all your traffic that had keyword and topic data.' %}</p>
+  <em>{% trans 'This report shows the <strong>top 20 keywords</strong> targeted by your flights and updates periodically. All previous days data is complete.' %}</em>
```

Comment:
```suggestion
  <em>{% trans 'This report shows the <strong>top 20 keywords</strong> targeted by your flights and updates daily. All previous days data is complete.' %}</em>
```

I think this is on all our reports, but they all updated daily now except revenue I think.